Makefile

```diff
@@ -31,3 +34,17 @@ test:
 # this prevents errors like "No rule to make target 'TestFaultInjection'" when you run "make test TestFaultInjection"
 Test%:
 	@:
+
+# this will install production operators
+test.presubmit.install:
+	scripts/ci/operator.sh create wait get_csv
+
+# this will delete production operators
+test.presubmit.cleanup:
+	scripts/ci/operator.sh delete delete_csv delete_cni
+
+# In an CI job, this will run tests on a remote OpenShift cluster
+test.presubmit:
```

Comment:
Can't this simply be called `presubmit` instead of `test.presubmit`?

But since my previous comment, I learned that this target is also going to be used to run interop tests. IF that's the case, then we need a different name. Maybe stick to what this target actually does (install operator and run the tests). So it could be called `test-with-operator` or something similar?

Reply:
Yes, I updated the target names. Now I use `presubmit` in this PR and `presubmit-install-operator`, `presubmit-cleanup-operator` for the set up/clean up.